src/main/java/org/fogbowcloud/saps/engine/core/archiver/exceptions/ArchiverException.java

```diff
@@ -0,0 +1,10 @@
+package org.fogbowcloud.saps.engine.core.archiver.exceptions;
+
+public class ArchiverException extends RuntimeException {
```

Comment:
exceptions are made to be handled. So, the type should help the handler part. `ArchiverException` means little.  A good test is: why not using the RuntimeException directly?

Reply:
Added fixme.